app/src/main/java/com/courseproject/tindar/usecases/editprofile/EditProfileDsResponseModel.java

```diff
@@ -2,16 +2,16 @@
 
 import java.util.Date;
 
-public class EditProfileResponseModel {
+public class EditProfileDsResponseModel {
 
     private final Date birthdate;
     private final String gender;
     private final String location;
     private final String profilePictureLink;
     private final String aboutMe;
 
-    public EditProfileResponseModel(Date birthdate, String gender, String location, String profilePictureLink,
-                                    String aboutMe) {
+    public EditProfileDsResponseModel(Date birthdate, String gender, String location, String profilePictureLink,
```

Comment:
might be better to have individual setters, setters for smaller variable groups, or pass objects instead of individual attributes so that we're not married to a certain set of parameters and parameter order.

Reply:
Let's discuss about other use cases, then figure out the design accordingly.